pkg/cmd/release/download/download.go

```diff
@@ -140,7 +140,7 @@ func downloadRun(opts *DownloadOptions) error {
 		return err
 	}
 
-	opts.IO.StartProgressIndicator()
+	opts.IO.StartProgressIndicatorWithLabel("Downloading ")
```

Comment:
Since we are already here making improvements what about changing this to: 
```suggestion
	opts.IO.StartProgressIndicatorWithLabel("Finding assets to download")
```